docker/docker.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+readonly REPOSITORY=ufcgsaps/catalog
+readonly USAGE="usage: docker.sh [-a] {build|push|publish|run} <TAG>"
+readonly MY_PATH=$(cd "$(dirname "${0}")" || { echo "For some reason, the path is not accessible"; exit 1; }; pwd )
```

Comment:
remove this "For some reason". It does not help at all.

Reply:
Done.